solutuins-1/src/test/java/solutions_1/solutuins_1/KattappaTest.java

```diff
@@ -0,0 +1,61 @@
+package solutions_1.solutuins_1;
+
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.Collection;
+import org.junit.Test;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * The Class PrimeStringTest.
+ */
+
+
+@RunWith(Parameterized.class)
+public class KattappaTest {
+
+	/**
+	 * Instantiating KattappaTest object
+	 */
+
+	KattapaReadyForBattle instance = new KattapaReadyForBattle();
+	
+	/**
+	 * Test case parameters
+	 */
+	
+	@Parameter(0)
+	public int inputArmySize;
+	@Parameter(1)
+	public Integer[] soldierWeapons;
+	@Parameter(2)
+	public String expected;
+
+	/**
+	 *  creates the test data
+	 * @return array list of data for each test case
+	 */
+	
+	@Parameters
+	public static Collection<Object[]> data() {
+		Object[][] data = new Object[][] { { 4,new Integer[]{11, 12, 13, 14},"NOT READY" }, { 3,new Integer[]{2, 3, 4},"READY FOR BATTLE" } };
+		return Arrays.asList(data);
+	}
+
+	/**
+	 * testing method tests the isReadyOrNot method.
+	 */
+
+	@Test
+	public void testing() { 

```

Comment:
The method name code be like kattapaReadyForBattle_Positive_Test

The verification can be simple like below
```
@Test
public void kattapaReadyForBattle_Positive_Test  () {
   assertEquals("NOT READY", kattapaBattleTest.isReadyForBattle(4, new Integer[]{11, 12, 13, 14}));
   assertEquals("READY FOR BATTLE", kattapaBattleTest.isReadyForBattle(4, new Integer[]{2, 3, 4));
}
```

Reply:
Similar to Positive Add Negative test cases

```
@Test
public void kattapaReadyForBattle_Negative_Test  () {
   assertEquals("Number of Soliders and weapon not matching ", kattapaBattleTest.isReadyForBattle(2, new Integer[]{11, 12, 13, 14}));
   assertEquals("Invalid input", kattapaBattleTest.isReadyForBattle(4, null);
}
```

Please include the junit result image.
And also include test code coverage image.

Reply:
I had included a screenshot. It's a .bmp file.